requirements/requirements.txt

```diff
@@ -105,7 +105,7 @@ pyyaml==6.0
     # via -r requirements/requirements.in
 requests==2.27.1
     # via -r requirements/requirements.in
-sentry-sdk==1.5.10
+sentry-sdk==1.6.0
```

Comment:
Strangely `pip-compile --output-file=requirements/requirements.txt requirements/requirements.in` doesn't capture this. It doesn't look like any of `sentry-sdk`s dependencies need to be updated, but it might be worth running the above pip-compile command to catch any other dependencies that can be incremented

Reply:
Should be sorted now. Ran this in python 3.8 and there were a few other changes but nothing massive.